scripts/migration.sh

```diff
@@ -34,6 +34,12 @@ printf "Generating migration boilerplate for patch level $NEW_LEVEL..."
 
 echo "SET NAMES utf8mb4 COLLATE utf8mb4_bin;\n" > "$FWD_SCHEMA"
 echo "-- TODO: Implement your forward migration here\n" >> "$FWD_SCHEMA"
+
+echo "SELECT value INTO @patchlevel FROM `dbMetadata` WHERE `name` = `schema-patch-level`;"
+echo "IF @patchlevel != $PREV_LEVEL THEN
+    SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 1643, MESSAGE_TEXT = 'Could not migrate to next patch';
```

Comment:
We need `MESSAGE_TEXT` to be as helpful as possible for the person running the migration when it fails. I don't think `'Could not migrate to next patch'` is the most helpful thing we can come up with here. To be really helpful, we should tell them precisely what the problem was so they know how to fix it.

Something along the lines of `'Missing migration detected'` would be clearer.